Mathlib/Logic/Equiv/Defs.lean

```diff
@@ -103,9 +103,10 @@ instance : EquivLike (α ≃ β) α β where
   coe_injective' e₁ e₂ h₁ h₂ := by cases e₁; cases e₂; congr
 
 /-- Helper instance when inference gets stuck on following the normal chain
-`EquivLike → EmbeddingLike → FunLike → CoeFun`. -/
-instance : FunLike (α ≃ β) α β :=
-  EmbeddingLike.toDFunLike
+`EquivLike → FunLike. -/
+instance : FunLike (α ≃ β) α β where
```

Comment:
No. I cannot "suggest" removing it though, because line 105 was not touched by this PR. Locally I have removed lines 105 to 110 and mathlib compiles fine.